### PR TITLE
[#169314231] request types generation from specs

### PR DESCRIPTION
**Pull request by alexgpeppe**
This PR aims to edit the OpenAPI specification file in order to allow the generation of request types and response decoders for the operations described in it.

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🐞 [#169314231](https://www.pivotaltracker.com/story/show/169314231): Il file di specifiche openapi non consente la generazione dei tipi per le operazoni

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 7d89d784294e9fca9e434f3344503bfa68111272
</p>

File: openapi.yml [hunk 1]
```diff
@@ -42,15 +42,14 @@ x-commons:
       $ref: https://teamdigitale.github.io/openapi/0.0.5/definitions.yaml#/responses/429TooManyRequests
     503:
       $ref: https://teamdigitale.github.io/openapi/0.0.5/definitions.yaml#/responses/503ServiceUnavailable
-    default:
```
Comment (ioggstream) [inline openapi.yml]: LGTM @gunzip `io-utils` both swagger and oas3 can't use `default` as a schema name. Is `default` a reserved typescript keyword?

File: yarn.lock [hunk 2]
```diff
@@ -2772,9 +2772,9 @@ italia-utils@^4.1.0:
     swagger-parser "^7.0.0"
     yargs "^11.1.0"
 
-italia-utils@aymen94/io-utils-build:
-  version "4.1.0"
-  resolved "https://codeload.github.com/aymen94/io-utils-build/tar.gz/4a59a9bc23301bd30d37542dccb3faf8394f618f"
+italia-utils@teamdigitale/io-utils-build-oas3:
+  version "4.1.1-beta.2"
```
Comment (ioggstream) [inline yarn.lock]: @gunzip I labeled the `OAS` package as `beta`. You should be able to edit https://github.com/teamdigitale/io-utils-build-oas3 as you please and eventually adjust that.

File: openapi.yml (hunk 1 above)
Comment (gunzip) [reply openapi.yml]: it is a reserved keywords in javascript (but it may be not related, it depends on the exact error you get using io-utils)

**Pull request merged**